lib/templates/videoAdvanced.mustache

```diff
@@ -38,9 +38,6 @@
 					{{#source}}
 					<a class="video-advanced-link video-advanced-source" href="{{ source }}">source</a>
 					{{/source}}
-					{{#downloadurl}}
-					<a class="video-advanced-link video-advanced-download res-icon" href="{{ downloadurl }}" title="download"></a>
```

Comment:
There's some CSS in _showImages.scss to style `.video-advanced-download` that can be removed if this is removed